### PR TITLE
Add Travis CI pipeline

### PR DESCRIPTION
**Pull request by mpaddle**
This PR transfers the CI tool to Travis CI: Github Actions is no longer supported on our account; most of us have used Travis CI and it's free for OSS projects.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -1,35 +0,0 @@
...
-
-    - name: Test with pytest
-      run: |
-        poetry run pytest --cov=./ --cov-report=xml
-    - name: Upload coverage to Codecov
-      uses: codecov/codecov-action@v1.0.0
-      with:
-        token: ${{secrets.CODECOV_TOKEN}}
-        file: ./coverage.xml
-        flags: unittests
```
Comment (mpaddle) [inline .github/workflows/ci.yml]: @vhemt Is this line important? I can't seem to recreate the behaviour in codecov with their configuration file 🤔

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -1,35 +0,0 @@
...
-        poetry run black --check .
-        poetry run mypy --ignore-missing-imports --disallow-untyped-defs .
-
-    - name: Test with pytest
-      run: |
-        poetry run pytest --cov=./ --cov-report=xml
-    - name: Upload coverage to Codecov
-      uses: codecov/codecov-action@v1.0.0
-      with:
-        token: ${{secrets.CODECOV_TOKEN}}
```
Comment (mpaddle) [inline .github/workflows/ci.yml]: @vhemt Because this is a public repository, this wasn't actually necessary AFAIK

Comment (vhemt) [reply .github/workflows/ci.yml]: ^ good point!

File: .github/workflows/ci.yml (hunk 1 above)
Comment (vhemt) [reply .github/workflows/ci.yml]: I have no idea, I think I just copied and pasted it :)

Comment (mpaddle) [reply .github/workflows/ci.yml]: Let's remove it then 👍

**Pull request merged**